src-vue/src/views/SettingsView.vue

```diff
@@ -12,6 +12,13 @@
                 <el-button icon="Folder" @click="updateGamePath"/>
             </template>
         </el-input>
+
+        <h3>Other:</h3>
+
+        <el-button type="primary" @click="openRepairView">
+            Open Repair options
+        </el-button>
```

Comment:
Not sure what to call the section the button is in so I went with "_Other_" for now ^^

Rendered:

![image](https://user-images.githubusercontent.com/40122905/204100337-80f12359-a5c4-456c-82db-259b56a7a034.png)